bt_project/obstacle_point[] obstacles
